src/components/StyleGuide/DesignTokens.stories.mdx

```diff
@@ -18,23 +18,39 @@ import { getCategory } from "../../utils/componentCategories";
 
 ## What are Design Tokens?
 
-Design tokens are the style values that define a design system. These values can include, but may not be limited to, spacing, color, typography, breakpoints and object styles. Ultimately, these values are used to construct and maintain the components that make up our system.
+Design tokens are the style values that define a design system. These values
+can include, but may not be limited to, spacing, color, typography, breakpoints
+and object styles. Ultimately, these values are used to construct and maintain
+the components that make up our system.
 
 ## Why Use Design Tokens?
 
-_"With design tokens, you can capture low-level values and then use them to create the styles for your product or app. You can maintain a scalable and consistent visual system for UI development."_ – Jina Bolton, Lead Designer for the Salesforce Design System
+_"With design tokens, you can capture low-level values and then use them to
+create the styles for your product or app. You can maintain a scalable and
+consistent visual system for UI development."_ - Jina Bolton, Lead Designer
+for the Salesforce Design System
 
-Design tokens are a single source of truth when it comes to styles. They allow an organization to more easily build multiple applications – web and/or native – that feel and look the same without slowing down the development team.
+Design tokens are a single source of truth when it comes to styles. They allow
+an organization to more easily build multiple applications - web and/or native
+
+- that feel and look the same without slowing down the development team.
```

Comment:
The hyphen adds this line as a bullet point.

Reply:
Ha!  Fixed.